.travis.yml

```diff
@@ -2,7 +2,6 @@ sudo: false
 dist: trusty
 
 language: php
-php: 7.3
```

Comment:
Since this configuration only applies to the `sniff` stage, we can just as well move it to the `sniff` stage as well.